spec/features/votes/user_votes_on_article_spec.rb

```diff
@@ -1,15 +1,17 @@
 require 'rails_helper'
+include EmailSpec::Helpers
+include EmailSpec::Matchers
```

Comment:
You can remove these two lines since they are in your spec_helper.rb. However, a better place for these is your rails_helper.rb, despite what the Apollo assignment says.